template/project.json

```diff
@@ -1,20 +1,30 @@
 {
-    "version": "1.0.0-*",
-    "compilationOptions": {
-        "emitEntryPoint": true
-    },
-
-    "compilerName": "fsc",
-    "compileFiles": [
-        "Program.fs"
-    ],
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "compilerName": "fsc",
+  "compileFiles": [
+    "Program.fs"
+  ],
+  "dependencies": {
+    "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160329",
+    "Microsoft.NETCore.App": {
+      "type": "platform",
```

Comment:
It appears the latest dotnet cli for ubuntu, version 1.0.1.001808, is failing because of this:
https://travis-ci.org/ctaggart/visualfsharp-docker/builds/120160110#L1354

```
The command "docker build -t docker.io/ctaggart/visualfsharp:$FSC_VERSION ." exited with 0.
1.46s$ docker run --rm -v $TRAVIS_BUILD_DIR/template:/root/template -w /root/template docker.io/ctaggart/visualfsharp:$FSC_VERSION /bin/bash -c './build.sh; exit $?'
error: TODO: unknown keyword platform
unknown keyword platform
unknown keyword platform
unknown keyword platform
```

https://github.com/dotnet/cli/blame/rel/1.0.0/src/dotnet/commands/dotnet-new/FSharp_Console/project.json.template#L13
cc @eerhardt